### DISCUSSION TITLE
Is it possible to allow registration with external identity provider only?

### DISCUSSION THREAD

**Discussion by tunhvn** [zitadel/zitadel#5782] [General]
I want to disable username password registration. Only allow registration with external identity provider (Google, Github, Microsoft...).
Is it possible to do this?

**Comment by hifabienne** [comment]
Hei. At the moment it is not.
But we have added this in the following issue, that when the general setting on the login policy should only trigger for local accounts #5654

**Reply by tunhvn** [reply]
Thank you, hope this feature released soon.
